rendermodules/rough_align/apply_rough_alignment_to_montages.py

```diff
@@ -70,10 +71,11 @@ def apply_rough_alignment(render,
                           scale,
                           Z,
                           apply_scale=False,
-                          consolidateTransforms=True):
+                          consolidateTransforms=True,
+                          remap_section_ids=False):
     z = Z[0] # z value from the montage stack - to be mapped to the newz values in lowres stack
     newz = Z[1] # z value in the lowres stack for this montage
-    
+    print(z, newz)
```

Comment:
substitute logger.info or debug for print() so we can see this in the workflow logs?

Reply:
removed it. This message will be a small fish out in the sea of logs... Not worth adding.